src/test/java/io/tackle/pathfinder/controllers/TranslatorResourceTest.java

```diff
@@ -0,0 +1,104 @@
+package io.tackle.pathfinder.controllers;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.common.ResourceArg;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.http.ContentType;
+import io.tackle.commons.testcontainers.KeycloakTestResource;
+import io.tackle.commons.testcontainers.PostgreSQLDatabaseTestResource;
+import io.tackle.commons.tests.SecuredResourceTest;
+import io.tackle.pathfinder.dto.AssessmentCategoryDto;
+import io.tackle.pathfinder.dto.AssessmentDto;
+import io.tackle.pathfinder.dto.AssessmentHeaderDto;
+import io.tackle.pathfinder.dto.AssessmentQuestionnaireDto;
+import io.tackle.pathfinder.dto.translator.TranslateDBDto;
+import io.tackle.pathfinder.model.questionnaire.Category;
+import io.tackle.pathfinder.model.questionnaire.Questionnaire;
+import io.tackle.pathfinder.services.AssessmentSvc;
+import lombok.extern.java.Log;
+import org.junit.jupiter.api.Test;
+
+import javax.inject.Inject;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@QuarkusTest
+@QuarkusTestResource(value = PostgreSQLDatabaseTestResource.class,
+        initArgs = {
+                @ResourceArg(name = PostgreSQLDatabaseTestResource.DB_NAME, value = "pathfinder_db"),
+                @ResourceArg(name = PostgreSQLDatabaseTestResource.USER, value = "pathfinder"),
+                @ResourceArg(name = PostgreSQLDatabaseTestResource.PASSWORD, value = "pathfinder")
+        }
+)
+@QuarkusTestResource(value = KeycloakTestResource.class,
+        initArgs = {
+                @ResourceArg(name = KeycloakTestResource.IMPORT_REALM_JSON_PATH, value = "keycloak/quarkus-realm.json"),
+                @ResourceArg(name = KeycloakTestResource.REALM_NAME, value = "quarkus")
+        }
+)
+@Log
+class TranslatorResourceTest extends SecuredResourceTest {
```

Comment:
Should we also add a native image test class  which extends these tests? It looks to me like it would work out of the box if we did.

Reply:
The current version of the test shouldn't work in Native mode since it relies on injecting services and consuming entities interacting directly with the JPA objects. For instance:
- https://github.com/konveyor/tackle-pathfinder/blob/b48dd2329dba5ef4f8d37ba38100e88ca4a69461/src/test/java/io/tackle/pathfinder/controllers/TranslatorResourceTest.java#L44
- https://github.com/konveyor/tackle-pathfinder/blob/b48dd2329dba5ef4f8d37ba38100e88ca4a69461/src/test/java/io/tackle/pathfinder/controllers/TranslatorResourceTest.java#L67